tests/integration/test_tls.py

```diff
@@ -50,7 +50,7 @@ async def test_mattermost_db(ops_test: OpsTest) -> None:
         )
         # Deploy TLS Certificates operator.
         config = {"generate-self-signed-certificates": "true", "ca-common-name": "Test CA"}
-        await ops_test.model.deploy(TLS_CERTIFICATES_APP_NAME, channel="edge", config=config)
+        await ops_test.model.deploy(TLS_CERTIFICATES_APP_NAME, channel="beta", config=config)
```

Comment:
BTW, https://charmhub.io/tls-certificates-operator channel beta has version 13, while edge is on 18.
Is Beta version good enough for us (just a question)?

Reply:
JFYI, reported the fresh issue: https://github.com/canonical/tls-certificates-operator/issues/23